test/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.test.cpp

```diff
@@ -207,24 +209,116 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Dynamics_Tabulated, Accessors)
 
 TEST_F(OpenSpaceToolkit_Astrodynamics_Dynamics_Tabulated, Getters)
 {
+    MatrixXd positionContributionsProfile_(defaultInstants_.getSize(), 3);
+    MatrixXd velocityContributionsProfile_(defaultInstants_.getSize(), 3);
+    MatrixXd massContributionsProfile_(defaultInstants_.getSize(), 1);
+    positionContributionsProfile_ << 1.0, 1.0, 1.0, 1.0, 2.0, 2.0, 2.0, 2.0, 3.0, 3.0, 3.0, 3.0;
```

Comment:
I see that you didn't overlap numbers between subsets to test better.
Still I would make entirely unique, since we are extracting subsets in a different order than the one they were originally stored in the TabulatedDynamics instance. E.g.:

```cpp
positionContributionsProfile_ << 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 11.0, 12.0;
velocityContributionsProfile_ << -1.0, -2.0, -3.0, -4.0, -5.0, -6.0, -7.0, -8.0, -9.0, -10.0, -11.0, -12.0;
massContributionsProfile_ << 0.001, 0.002, 0.003, 0.004;
```

Reply:
good point